docs/contribution.md

```
@@ -0,0 +1,43 @@
+Examples: `feat-add-instructor-domain`, `fix-login-bug`
+
+## 3. Commit Messages
+- **Optional**: follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).
+- In short:
+  ```
+  feat: add new instructor endpoint
+  fix: correct email validation bug
+  chore: update dependencies
+  ```
+
+## 4. Pull Requests
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)